core/src/main/scala/cats/effect/Console.scala

```diff
@@ -74,4 +74,8 @@ object Console {
     * Default instance for `Console[IO]` that prints to standard input/output streams.
     */
   val io: Console[IO] = SyncConsole.stdio[IO]
+
+  object implicits {
+    implicit val instance: Console[IO] = io
```

Comment:
I'd consider renaming this to `io` too, in a similar fashion to Scala's global ExecutionContext. Or `ioConsole`, which would be even better because implicits have kind of a global namespace and `io` will potentially clash very easily

Reply:
:+1:

Reply:
I renamed the val to `ioConsole`. 👍